google-cloud-clients/google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta1/BigQueryStorageClient.java

```diff
@@ -501,21 +491,24 @@ public final void finalizeStream(FinalizeStreamRequest request) {
    *
    * <pre><code>
    * try (BigQueryStorageClient bigQueryStorageClient = BigQueryStorageClient.create()) {
-   *   Stream originalStream = Stream.newBuilder().build();
+   *   Stream originalStream = getStream();
    *   SplitReadStreamResponse response = bigQueryStorageClient.splitReadStream(originalStream);
    * }
    * </code></pre>
    *
    * @param originalStream Stream to split.
    * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   * @deprecated Use {@link BigQueryStorageClient#splitReadStream(SplitReadStreamRequest)}.
    */
   public final SplitReadStreamResponse splitReadStream(Stream originalStream) {
     SplitReadStreamRequest request =
-        SplitReadStreamRequest.newBuilder().setOriginalStream(originalStream).build();
+        SplitReadStreamRequest.newBuilder()
+            .setOriginalStream(originalStream)
+            .setFraction(0.5f)
```

Comment:
I would suggest to move the `0.5f`  value to a constant `STREAM_SPLIT_FRACTION` and use it in all places!